diff --git a/main.lua b/main.lua
index 356b132..84b936a 100644
--- a/main.lua
+++ b/main.lua
@@ -288,8 +288,8 @@ return {
 
 		if cmd == "g" then
 			if direction == "g" then
-				ya.mgr_emit("arrow", { "top" })
-				ya.mgr_emit("arrow", { lines - 1 })
+				ya.emit("arrow", { "top" })
+				ya.emit("arrow", { lines - 1 })
 				render_clear()
 				return
 			elseif direction == "j" then
@@ -297,7 +297,7 @@ return {
 			elseif direction == "k" then
 				cmd = "k"
 			elseif direction == "t" then
-				ya.mgr_emit("tab_switch", { lines - 1 })
+				ya.emit("tab_switch", { lines - 1 })
 				render_clear()
 				return
 			else
@@ -308,66 +308,66 @@ return {
 		end
 
 		if cmd == "j" then
-			ya.mgr_emit("arrow", { lines })
+			ya.emit("arrow", { lines })
 		elseif cmd == "k" then
-			ya.mgr_emit("arrow", { -lines })
+			ya.emit("arrow", { -lines })
 		elseif cmd == "h" then
 			for _ = 1, lines do
-				ya.mgr_emit("leave", {})
+				ya.emit("leave", {})
 			end
 		elseif cmd == "l" then
 			for _ = 1, lines do
-				ya.mgr_emit("enter", {})
+				ya.emit("enter", {})
 				local file_idx = get_cache_or_first_dir()
 				if file_idx then
-					ya.mgr_emit("arrow", { "top" })
-					ya.mgr_emit("arrow", { file_idx })
+					ya.emit("arrow", { "top" })
+					ya.emit("arrow", { file_idx })
 				end
 			end
 		elseif is_tab_command(cmd) then
 			if cmd == "t" then
 				for _ = 1, lines do
-					ya.mgr_emit("tab_create", {})
+					ya.emit("tab_create", {})
 				end
 			elseif cmd == "H" then
-				ya.mgr_emit("tab_switch", { -lines, relative = true })
+				ya.emit("tab_switch", { -lines, relative = true })
 			elseif cmd == "L" then
-				ya.mgr_emit("tab_switch", { lines, relative = true })
+				ya.emit("tab_switch", { lines, relative = true })
 			elseif cmd == "w" then
-				ya.mgr_emit("tab_close", { lines - 1 })
+				ya.emit("tab_close", { lines - 1 })
 			elseif cmd == "W" then
 				local curr_tab = get_active_tab()
 				local del_tab = curr_tab + lines - 1
 				for _ = curr_tab, del_tab do
-					ya.mgr_emit("tab_close", { curr_tab - 1 })
+					ya.emit("tab_close", { curr_tab - 1 })
 				end
-				ya.mgr_emit("tab_switch", { curr_tab - 1 })
+				ya.emit("tab_switch", { curr_tab - 1 })
 			elseif cmd == "<" then
-				ya.mgr_emit("tab_swap", { -lines })
+				ya.emit("tab_swap", { -lines })
 			elseif cmd == ">" then
-				ya.mgr_emit("tab_swap", { lines })
+				ya.emit("tab_swap", { lines })
 			elseif cmd == "~" then
 				local jump = lines - get_active_tab()
-				ya.mgr_emit("tab_swap", { jump })
+				ya.emit("tab_swap", { jump })
 			end
 		else
-			ya.mgr_emit("visual_mode", {})
+			ya.emit("visual_mode", {})
 			-- invert direction when user specifies it
 			if direction == "k" then
-				ya.mgr_emit("arrow", { -lines })
+				ya.emit("arrow", { -lines })
 			elseif direction == "j" then
-				ya.mgr_emit("arrow", { lines })
+				ya.emit("arrow", { lines })
 			else
-				ya.mgr_emit("arrow", { lines - 1 })
+				ya.emit("arrow", { lines - 1 })
 			end
-			ya.mgr_emit("escape", {})
+			ya.emit("escape", {})
 
 			if cmd == "d" then
-				ya.mgr_emit("remove", {})
+				ya.emit("remove", {})
 			elseif cmd == "y" then
-				ya.mgr_emit("yank", {})
+				ya.emit("yank", {})
 			elseif cmd == "x" then
-				ya.mgr_emit("yank", { cut = true })
+				ya.emit("yank", { cut = true })
 			end
 		end
 
